### DISCUSSION TITLE
Android app failure: Type android.support.v4.os.IResultReceiver$Stub$Proxy is defined multiple times

### DISCUSSION THREAD

**Discussion by AArnott** [AvaloniaUI/Avalonia#13766] [Q&A]
After creating an Avalonia app (and developing it on Windows desktop for a while), I turned my attention to the Android app, which fails to build with this error:

> 1>MSBUILD : java.exe error JAVA0000: Type android.support.v4.os.IResultReceiver$Stub$Proxy is defined multiple times: C:\Users\andre\source\repos\Nerdbank.Cryptocurrencies\obj\src\X.Android\ARM64\Debug\net8.0-android\lp\49\jl\classes.jar:android/support/v4/os/IResultReceiver$Stub$Proxy.class, C:\Users\andre\source\repos\Nerdbank.Cryptocurrencies\obj\src\X.Android\ARM64\Debug\net8.0-android\lp\6\jl\bin\classes.jar:android/support/v4/os/IResultReceiver$Stub$Proxy.class

Any idea why?

I added `<PackageReference Include="Xamarin.AndroidX.Annotation" />` to the Android project file in order to resolve this warning:

> JAVAC : error : unknown enum constant Scope.LIBRARY_GROUP_PREFIX 

But the Stub$Proxy error occurs before and after this change. Other than this change, I believe the Android project is essentially identical to what the avalonia project template created.

**Comment by maxkatz6** [comment]
Why does every major SDK update have to break something about mobile workloads? It really upsets me.
Either way, it doesn't seem to be related to the Avalonia directly. But most likely can be caused by some dependency version.

**Reply by maxkatz6** [reply]
Can you try the latest nighly builds? We have updated SkiaSharp there, and AndroidX should be already up to date there.
